pkg/k8s/apis/cilium.io/client/crds/v2alpha1/ciliumenvoyconfigs.yaml

```diff
@@ -0,0 +1,115 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.6.2
+  creationTimestamp: null
+  name: ciliumenvoyconfigs.cilium.io
+spec:
+  group: cilium.io
+  names:
+    categories:
+    - cilium
+    kind: CiliumEnvoyConfig
+    listKind: CiliumEnvoyConfigList
+    plural: ciliumenvoyconfigs
+    shortNames:
+    - cec
+    singular: ciliumenvoyconfig
+  scope: Cluster
+  versions:
+  - additionalPrinterColumns:
+    - description: The age of the identity
+      jsonPath: .metadata.creationTimestamp
+      name: Age
+      type: date
+    name: v2alpha1
+    schema:
+      openAPIV3Schema:
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            properties:
+              backendServices:
```

Comment:
It took me awhile to understand what `backendServices` is and how it differs from `services`. I find the name a bit misleading. Maybe `servicesWhichBypassEnvoyLB` or so could be a better term.

Reply:
All services bypass L7 LBs by default. I'm open for better names in the CRD API, but it would need to be more like `servicesWhoseBackendsAreSyncedToEnvoyWithoutFrontendBeingRedirected` I have some follow-on work queued up in this area, so we can come back to naming also after this one merges.

Reply:
To be honest, I still struggle to understand the name. The first time I saw it I couldn't figure out what this was and the description wasn't very descriptive either.  Perhaps a better explanation could be given in the description? But ideally another name too.

Reply:
Updated the description:
```
 	// BackendServices specifies Kubernetes services whose backends
-	// are automatically synced to Envoy usign EDS.
+	// are automatically synced to Envoy using EDS.  Traffic for these
+	// services is not forwarded to an Envoy listener. This allows an
+	// Envoy listener load balance traffic to these backends while
+	// normal Cilium service load balancing takes care of balancing
+	// traffic for these services at the same time.
```

Reply:
I did not come up with a better name yet.

Reply:
thanks fine, the description addition is already very helpful, thank you!